nixos/modules/services/x11/desktop-managers/surf-display.nix

```diff
@@ -0,0 +1,140 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.surf-display;
+
+  surfDisplayConf = ''
+    # Surf Kiosk Display: Wrap around surf browser and turn your
+    # system into a browser screen in KIOSK-mode.
+
+    # default download URI for all display screens if not configured individually
+    DEFAULT_WWW_URI="${cfg.defaultWwwUri}"
+
+    # Enforce fixed resolution for all displays (default: not set):
+    ${if cfg.defaultResolution == "" then ''
```

Comment:
So this option and the one below it say `default: not set` so maybe we can get away with not even having an option?

Reply:
So people use extraConfig for those instead, if they want them. That could work yes.